<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RIOMDebug</class>
 <widget class="QWidget" name="RIOMDebug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: white;
	border: 2px groove, gray;
    border-color: gray;
	font: 20px, &quot;SimHei&quot;;	
}

QWidget
{
	background-color: black;
}

QPushButton
{
	font: 20px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
	border-radius: 5px;
}</string>
  </property>
  <widget class="QLabel" name="label_CAR4_DI3">
   <property name="geometry">
    <rect>
     <x>787</x>
     <y>165</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR3_DI4">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>194</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR2_AX">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>252</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR3_AX">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>252</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR4_DI4">
   <property name="geometry">
    <rect>
     <x>787</x>
     <y>194</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR3_GW">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>281</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR2_DI4">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>194</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR4_AX">
   <property name="geometry">
    <rect>
     <x>787</x>
     <y>252</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_DI4">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>194</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>DI4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR1_DI1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>107</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_GW">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>281</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>IO-GW</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR3_DI2">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>136</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_DI2">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>136</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>DI2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR1_DI4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>194</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>550</y>
     <width>98</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR4_DI2">
   <property name="geometry">
    <rect>
     <x>787</x>
     <y>136</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR1_AX">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>252</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR4_DI1">
   <property name="geometry">
    <rect>
     <x>787</x>
     <y>107</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR1_GW">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>281</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR2_DO">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>223</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR3_DI1">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>107</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR1_DI3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>165</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_AX">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>252</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>AX</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR4_DO">
   <property name="geometry">
    <rect>
     <x>787</x>
     <y>223</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR4_GW">
   <property name="geometry">
    <rect>
     <x>787</x>
     <y>281</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_DO">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>223</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>DO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_DI3">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>165</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>DI3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR2_DI1">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>107</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR2_DI2">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>136</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_DI1">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>107</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>DI1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR2_GW">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>281</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR3_DO">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>223</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR3_DI3">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>165</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR1_DI2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>136</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR1_DO">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>223</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR2_DI3">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>165</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR2_LAT">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>310</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_GW_2">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>339</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>RLD</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR4_RLD">
   <property name="geometry">
    <rect>
     <x>787</x>
     <y>339</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR1_LAT">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR3_RLD">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>339</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR1_RLD">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>339</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR4_LAT">
   <property name="geometry">
    <rect>
     <x>787</x>
     <y>310</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR2_RLD">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>339</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR3_LAT">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>310</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_AX_2">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>310</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>LAT</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
